kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/configuration/NMSettingsConverter.java

```diff
@@ -98,6 +107,108 @@ public static Map<String, Map<String, Variant<?>>> buildSettings(NetworkProperti
         return newConnectionSettings;
     }
 
+    public static Map<String, Variant<?>> build8021xSettings(NetworkProperties props, String deviceId) {
```

Comment:
Rename this method for consistency. We reserved `build*` for the top level public methods, `create*` for the private ones called by the `build*` ones.

```suggestion
    public static Map<String, Variant<?>> create8021xSettings(NetworkProperties props, String deviceId) {
```